spec/wor/push/notifications/validators/push_notifications_validator_spec.rb

```diff
@@ -90,6 +110,16 @@
       end
     end
 
+    context 'when the model is nil' do
+      let(:model) { nil }
+      let(:message_content) { { other_field: 'some information' } }
+
+      it 'raises runtime error with a descriptive message' do
+        expect { validate_send_message }.to raise_error(RuntimeError,
```

Comment:
RuntimeError? Can't we write some custom exception for this? otherwise it's hard to catch it, think that the final user can't catch RuntimeError because he will be catchi most of the ruby errors and not only our exception